.flintci.yml

```diff
@@ -1 +0,0 @@
-project/.flintci.yml
```

Comment:
I think we could maintain those symlinks but with the new location right?

Reply:
What about using the same process than other Sonata bundle and use the DispatchCommand to update these files instead ?

Reply:
I can still reuse a symbolic link (just need to look at the command online ^^'), but this won't work the day this file will be change to `.flintci.yml.twig`

Reply:
> What about using the same process than other Sonata bundle and use the DispatchCommand to update these files instead ?

Well we could do it, but it will be a bit problematic because the dispatcher dispatch a lot of files and we only use those 2 here.

I would say to maintain the symlinks for now (to have dev-kit) working again and finish the orm master branch with the admin) and think on how to improve this later.

And, if we have just 2 files, when they become twig, we can just duplicate them and not worry about adding more code.

Reply:
Done.
I did the same for `.php_cs.dist` and `.editorconfig`.